implementations/rust/ockam/ockam_api/src/config/cli.rs

```diff
@@ -175,3 +176,26 @@ impl ConfigValues for AuthoritiesConfig {
         Self::default()
     }
 }
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct Authority {
+    identity: HexByteVec,
```

Comment:
Ideally we would use `PublicIdentity` here, but the type does not implement `Serialize` or `Deserialize`. It is a bit sad, because in `ockam_command` we already have to import the bytes as a `PublicIdentity` in order to get the `IdentityIdentifier` only to serialize it again and converting it back yet again in `AuthoritiesConfig::to_public_identities`.